test/auth.spec.js

```diff
@@ -61,7 +61,7 @@ describe('client authentication', function () {
                 });
                 var tmp = client.command_queue.get(0).callback;
                 client.command_queue.get(0).callback = function (err, res) {
-                    client.auth = function (pass, callback) {
+                    client.auth = function (pass, user, callback) {
```

Comment:
@leibale shouldn't we keep the original test too? for backward?

Reply:
@gkorland overriding a function on the client is not part of the API..